template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+# Template deliverables
+
+This document explains the different documentation deliverables that template authors are required to write as part of each template set. First, read our Template Contributing Guide to understand the larger context for these deliverables.
+
+## Table of contents
+
+- [Overview](#overview)
+- [Raw template](#raw-template)
+- [Template guide](#template-guide)
+- [Before you start guide (optional)](#before-you-start-guide-optional)
+- [Template example (optional)](#template-example-optional)
+
+
+## Overview
+
+Each template set should have the following components:
+
+<table>
+  <tr>
+    <th>Deliverable</th>
+    <th>Description and purpose</th>
+    <th>Required?</th>
+  </tr>
+  <tr>
+    <td>Raw template</td>
+    <td>The raw template is the basic template for the type of document you are creating. It provides a rough outline of the suggested content and a few embedded writing tips for how to fill in the different sections of the template.</td>

```

Comment:
Calling this the "raw template" or the "basic template" unintentionally implies there might be other templates in the template sets. There is just one, and this is it. It is the template itself. Perhaps this should read "The template file itself contains a suggested content structure, with fields that a user can replace to generate a document of this type." Something like that.

Reply:
Sure.

Reply:
I've adopted this suggestion into the draft. Resolving.